core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceService.scala

```diff
@@ -252,7 +260,7 @@ class MultiCloudWorkspaceService(override val ctx: RawlsRequestContext,
       // to avoid naming conflicts - we'll erase it should the clone request to WSM fail.
       newWorkspace <- createNewWorkspaceRecord()
 
-      _ <- (for {
+      containerCloneResult <- (for {
```

Comment:
Might be worth pulling this for comp out now that it's more than just the cloneWorkspace request. Nested for comps can be hard to read

Reply:
The goal was to avoid duplicating the error recovery logic (where we delete the Rawls workspace record), so I think it's good to keep it as-is.